changelog.md

```diff
@@ -1,3 +1,252 @@
+
+ <a name="6.0.0-beta.0"></a>
+# 6.0.0-beta.0 (2020-03-11)
+[Full Changelog](https://github.com/GoogleChrome/lighthouse/compare/v5.6.0...v6.0.0-beta.0)
+
+(A proper thanks for our new contributors will be in the final 6.0 release notes.)
+
+This beta release will not ship in Chrome DevTools, but we'll ship the final 6.0 release.
+
+## Notable Changes
+* Brand new metrics and performance score calculation (seeking feedback)
+* Lots more, complete release notes brewing..
+
+An unedited changelog follows…
+
+
+## New Audits
+
+* pwa: maskable icon audit ([#10370](https://github.com/GoogleChrome/lighthouse/pull/10370))
+* add cumulative layout shift metric ([a8a0f8f](https://github.com/GoogleChrome/lighthouse/commit/a8a0f8f))
+* add charset declaration audit ([#10284](https://github.com/GoogleChrome/lighthouse/pull/10284))
+* add new accessibility audits ([#9798](https://github.com/GoogleChrome/lighthouse/pull/9798))
+
+## Core
+
+* artifacts: add property attribute to MetaElements ([#9978](https://github.com/GoogleChrome/lighthouse/pull/9978))
+* audits: add handling of 'incomplete' results from axe-core ([#10072](https://github.com/GoogleChrome/lighthouse/pull/10072))
+* audits: Add more keywords to blocklist ([#9986](https://github.com/GoogleChrome/lighthouse/pull/9986))
+* config: add loadFailureMode warn ([#9987](https://github.com/GoogleChrome/lighthouse/pull/9987))
+* config: enable unused-javascript by default ([#9854](https://github.com/GoogleChrome/lighthouse/pull/9854))
+* duplicated-javascript: new audit ([ea9d226](https://github.com/GoogleChrome/lighthouse/commit/ea9d226))
+* emulation: bump emulated chrome UA to m80 ([#9967](https://github.com/GoogleChrome/lighthouse/pull/9967))
```

Comment:
we're going to want to give these commit notes the same treatment we do in all other releases, but we can defer that to the actual 6.0 notes.